lib/polyfill/media_capabilities.js

```diff
@@ -27,7 +27,12 @@ shaka.polyfill.MediaCapabilities = class {
     // Since MediaCapabilities is not fully supported on Chromecast yet, we
     // should always install polyfill for Chromecast.
     // TODO: re-evaluate MediaCapabilities in the future versions of Chromecast.
-    if (!shaka.util.Platform.isChromecast() && navigator.mediaCapabilities) {
+    // Since MediaCapabilities is not fully supported on Apple browsers yet, we
+    // should always install polyfill for Apple browsers.
+    // TODO: re-evaluate MediaCapabilities in the future versions of Apple Browsers.
```

Comment:
This line is too long. We maintain a column width of 80 characters. The last word will need to be moved to another line.

Reply:
Done!